src/components/footer/__tests__/__snapshots__/Footer.test.js.snap

```diff
@@ -13,17 +13,17 @@ exports[`renders the footer element 1`] = `
     <button
       class="button tertiary small rounded capitalize"
     >
-      Terms & Conditions
+      footer.legal.terms
```

Comment:
Instead of testing for the actual test, we are now testing the keys. Reason for this is in our test the `I18nProvider` is not defined, I feel like we should not add it. Also.... I think we should remove snapshot testing... but that's another topic xD